server/instrumentation/pre_workflow_hook.go

```diff
@@ -0,0 +1,27 @@
+package instrumentation
+
+import (
+	"github.com/runatlantis/atlantis/server/events"
+	"github.com/runatlantis/atlantis/server/events/command"
+	"github.com/runatlantis/atlantis/server/events/metrics"
+)
+
+type PreWorkflowHookRunner struct {
+	events.PreWorkflowHooksCommandRunner
+}
+
+func (r *PreWorkflowHookRunner) RunPreHooks(ctx *command.Context) error {
+	scope := ctx.Scope.SubScope("pre-workflow-hook")
+
+	executionSuccess := scope.Counter(metrics.ExecutionSuccessMetric)
+	executionError := scope.Counter(metrics.ExecutionErrorMetric)
+
+	err := r.PreWorkflowHooksCommandRunner.RunPreHooks(ctx)
+	if err != nil {
+		executionError.Inc(1)
+		return err
+	}
+
```

Comment:
I would add a success info log here (using my new logger, see instrumented_client) and remove the other random success log that gets emitted.

Reply:
added success log, but not sure what do you mean about new logger.